mypy_baseline/commands/_filter.py

```diff
@@ -37,12 +37,13 @@ def run(self) -> int:
                 unresolved_errors.append(error)
 
         fixed_errors: list[Error] = []
-        for line in baseline:
-            error = Error.new(line)
-            if error is None:
-                print(f'invalid baseline, cannot parse line: {line}')
-                return 1
-            fixed_errors.append(error)
+        if baseline != [""]: # When there are no errors, baseline is empty
```

Comment:
flake8 fails here on CI. Please, make sure CI passes. You can run everything locally with `task all`.

Reply:
Fixed